sdk/storage/src/table/requests/create_table_builder.rs

```diff
@@ -0,0 +1,67 @@
+use crate::table::prelude::*;
+use crate::table::responses::*;
+use azure_core::headers::add_optional_header;
+use azure_core::prelude::*;
+use http::method::Method;
+use http::status::StatusCode;
+use std::convert::TryInto;
+
+#[derive(Debug, Clone)]
+pub struct CreateTableBuilder<'a> {
+    table_client: &'a TableClient,
+    client_request_id: Option<ClientRequestId<'a>>,
+}
+
+impl<'a> CreateTableBuilder<'a> {
+    pub(crate) fn new(table_client: &'a TableClient) -> Self {
+        Self {
+            table_client,
+            client_request_id: None,
+        }
+    }
+
+    setters! {
+        client_request_id: ClientRequestId<'a> => Some(client_request_id),
+    }
+
+    pub async fn execute(
+        &self,
+    ) -> Result<CreateTableResponse, Box<dyn std::error::Error + Sync + Send>> {
+        let url = self.table_client.url();
+        debug!("url = {}", url);
```

Comment:
Nit: If we're going to use `debug!` we'll probably want more descriptive log messages.